indico/web/flask/util.py

```diff
@@ -321,6 +321,21 @@ def send_file(name, path_or_fd, mimetype, last_modified=None, no_cache=True, inl
     return rv
 
 
+def find_url_endpoint(url):
```

Comment:
heh, i think we used to have a very similar function in indico/web/flask/util.py until some months ago when i removed it as it had become obsolete ;)